kf_lib_data_ingest/target_apis/kids_first.py

```diff
@@ -182,8 +182,8 @@
             }
         ],
         'properties': {
-            'external_sample_id': CONCEPT.BIOSPECIMEN.UNIQUE_KEY,
-            'external_aliquot_id': CONCEPT.BIOSPECIMEN.ALIQUOT_ID,
+            'external_sample_id': CONCEPT.BIOSPECIMEN_GROUP.UNIQUE_KEY,
+            'external_aliquot_id': CONCEPT.BIOSPECIMEN.UNIQUE_KEY,
```

Comment:
This will affect the portal display. I don't know if that's acceptable.

Reply:
Lol ok so should I back out the changes I just made for BIOSPECIMENs UNIQUE KEY to be a compound of its sample group and aliquot?

Reply:
no. I think if allison says it shouldn't be on the portal then we change these to be the individual IDs

Reply:
I mean maybe we should just make them the individual ids since they are each capturing different ids?

Reply:
I guess we have this open question of exactly what the aliquot ID should contain. Should it be just the subcomponent or should it be a fully qualified ID more specific than the sample ID? TARGET data from GDC uses the latter.

Reply:
@allisonheath second question. should Aliquot ID always be a fully qualified identifier including the sample group component (like what GDC has with the TCGA barcodes)?

Reply:
Including extra components such as the same sample group into Aliquot ID seems somehow redundant (wouldn't hurt though). And other systems/applications such as PedcBioPortal may need to do additional data processing to parse out Aliquot ID from the compound ID.